index.html

```diff
@@ -22,16 +22,21 @@
       <nav class="mobile-menu">
         <h1 class="logo">My Logo</h1>
```

Comment:
Looks good, but please make sure to follow the exact Figma template. In this case you need to use the same font.

![image](https://user-images.githubusercontent.com/34779928/194646372-e0c3ed3d-aac7-4ff1-8ff4-4d4013859b86.png)